pages/common/bioradtopgm.md

```diff
@@ -0,0 +1,12 @@
+# bioradtopgm
+
+> Convert a Biorad confocal file into a PGM file.
+> More information: <https://netpbm.sourceforge.net/doc/bioradtopgm.html>.
+
+- Read a Biorad confocal file as input and produce a PGM image as output:
+
+`bioradtopgm {{path/to/input_file}} > {{path/to/output_file.pgm}}`
```

Comment:
```suggestion
- Read a Biorad confocal file and store the n'th image contained in it to as a PGM file:

`bioradtopgm -image {{n}} {{path/to/input_file}} > {{path/to/output_file.pgm}}`
```

The way I read the documentation, this is how the program converts between biorad and pgm format. See the comments on the `-image` option.

Reply:
Sure, I'll do that